src/dotnet-interactive.Tests/FormatterConfigurationTests.cs

```diff
@@ -49,21 +41,6 @@ public void LatexString_type_is_formatted()
         formattedValue.Value.Should().Be(@"F(k) = \int_{-\infty}^{\infty} f(x) e^{2\pi i k} dx");
     }
 
-    [Fact]
-    public void MathString_type_is_formatted()
```

Comment:
`MathString` is now obsolete - but should this test be preserved until we remove the `MathString` type?

Reply:
It's not worth the trouble because the implementation is trivial and essentially identical to `LaTeXString` and neither has been changed in years.